jbpm-flow-builder/src/main/resources/META-INF/kie.conf

```diff
@@ -1 +1,5 @@
-org.drools.compiler.compiler.ProcessBuilderFactoryService=org.jbpm.process.builder.ProcessBuilderFactoryServiceImpl
\ No newline at end of file
+org.drools.compiler.compiler.ProcessBuilderFactoryService=org.jbpm.process.builder.ProcessBuilderFactoryServiceImpl
+org.kie.api.internal.assembler.KieAssemblers = +org.jbpm.assembler.BPMN2AssemblerService
+org.kie.api.internal.weaver.KieWeavers = +org.jbpm.weaver.BPMN2WeaverService
+org.kie.api.internal.assembler.KieAssemblers = +org.jbpm.assembler.DRFAssemblerService
+org.kie.api.internal.weaver.KieWeavers = +org.jbpm.weaver.DRFWeaverService
```

Comment:
don't we need CMMN WeaverService as well?

Reply:
it's in `jbpm-case-mgmt/jbpm-case-mgmt-cmmn/src/main/resources/META-INF/kie.conf `